### PR TITLE
kde-apps: Push down *l10n deps into all meta packages to improve L10N coverage

### PR DESCRIPTION
**Pull request by a17r**
An array of ${PN} which lists kde-apps/ L10N packages to add to RDEPEND.
Currently known L10N packages: kde-l10n, kde4-l10n, kdepim-l10n, ktp-l10n
If non-empty, add nls to IUSE to control these dependencies.

EDIT: Not sure about that one yet.

### CODE REVIEW

File: eclass/kde5-meta-pkg.eclass [hunk 1]
```diff
@@ -14,6 +14,21 @@ _KDE5_META_PKG_ECLASS=1
 
 inherit kde5-functions
 
+# @ECLASS-VARIABLE: KDE_L10N_DEPS
+# @DESCRIPTION:
+# An array of ${PV} which lists kde-apps/ L10N packages to add to RDEPEND.
```
Comment (Arfrever) [inline eclass/kde5-meta-pkg.eclass]: s/${PV}/${PN}/

File: eclass/kde5-meta-pkg.eclass [hunk 2]
```diff
@@ -14,6 +14,21 @@ _KDE5_META_PKG_ECLASS=1
...
+# Currently known L10N packages are: kde-l10n, kde4-l10n, kdepim-l10n, ktp-l10n
+# If non-empty, add nls to IUSE to control these dependencies.
+if [[ -n ${KDE_L10N_DEPS} ]]; then
```
Comment (Arfrever) [inline eclass/kde5-meta-pkg.eclass]: if [[ ${#KDE_L10N_DEPS[@]} -gt 0 ]]; then

File: eclass/kde5-meta-pkg.eclass [hunk 3]
```diff
@@ -14,6 +14,21 @@ _KDE5_META_PKG_ECLASS=1
...
+	IUSE+=" nls"
+	local _L10N_MINIMAL _L10N_DEP
```
Comment (Arfrever) [inline eclass/kde5-meta-pkg.eclass]: It is global scope, so 'local' should not be used. This line can be dropped. 'unset' is already properly used below.

**Comment by a17r** [comment]
eclass--
ebuilds++

It makes more sense like that.

**Comment by kensington** [comment]
Thanks, picked as 73b18a5ab682a916efaee9b3fb075cd47439ddec...85c1137a43e7ec6ccf9604bccedf92ef7f547f38.

**Pull request closed without merging**